config/development.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The main reason for creating such file is not to expose production configs (Analytics especially). Divide it into several files will not resolve this, so I encourage to encrypt and decrypt during Travis build. See: https://docs.travis-ci.com/user/encrypting-files/

Reply:
Didn't get the idea

Reply:
People fork this repository or Ukrainian one and in most cases they don't change GA code, as a result, we get incorrect data. Same for mailchimp

Reply:
We can just push to master empty config files just with skeleton, people have to see how and what to fill in config

Reply:
We can just push to master empty config files just with skeleton, people have to see how and what to fill in config

Reply:
If use encryption, we'll have to maintain super_secret.txt, which can't be pushed to repo

Reply:
""There is a report of this function not working on a local Windows machine. Please use the WSL (Windows Subsystem for Linux) or a Linux or OS X machine."

Reply:
You upload the only dev.json filled. prod will be empty. After that, they fill with their sensitive data, add this file to gitigone and push only encrypted config. As a result, it works locally and on travis, data secured

Reply:
How are we going to maintain prod configs? Storing them in gitignore not a good solution, honestly. 
That's might be another discussion, but I think it can be a reason to have DevFest Ukraine repo private, with comitted prod config